### PR TITLE
import BinaryQuadraticModel

### PR DESCRIPTION
**Pull request by mhlr**
loading from json is currently failing with

```
bqm = json.loads(open(s).read(), cls=DimodDecoder)
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
/tmp/ipykernel_31901/580148923.py in <module>
      1 s = '/home/daniel/Downloads/bqm_cmi_german-credit_k14.json'
----> 2 bqm = json.loads(open(s).read(), cls=DimodDecoder)
      3 
      4 #s = '/home/daniel/hss-overview-benchmarks/qubos/g000283.qubo'
      5 #s = '/home/daniel/hss-overview-benchmarks/qubos/g001327.qubo'

~/anaconda3/lib/python3.8/json/__init__.py in loads(s, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    368     if parse_constant is not None:
    369         kw['parse_constant'] = parse_constant
--> 370     return cls(**kw).decode(s)

~/anaconda3/lib/python3.8/json/decoder.py in decode(self, s, _w)
    335 
    336         """
--> 337         obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    338         end = _w(s, end).end()
    339         if end != len(s):

~/anaconda3/lib/python3.8/json/decoder.py in raw_decode(self, s, idx)
    351         """
    352         try:
--> 353             obj, end = self.scan_once(s, idx)
    354         except StopIteration as err:
    355             raise JSONDecodeError("Expecting value", s, err.value) from None

~/anaconda3/lib/python3.8/site-packages/dimod/serialization/json.py in dimod_object_hook(obj)
     82         # in the future we could handle subtypes but right now we just have the
     83         # one
---> 84         return BinaryQuadraticModel.from_serializable(obj)
     85     return obj
     86 

NameError: name 'BinaryQuadraticModel' is not defined

```

### CODE REVIEW

**Comment by arcondello** [comment]
Can you add a test and a release note?

**Comment by mhlr** [comment]
> Can you add a test and a release note?

Do you know why the current tests are not catching this?
There are tests with `json.loads(..., cls=DimodDecoder)`
I would have expected them to be hitting this already

File: releasenotes/notes/fix-dimod-importer-bqm-import-8c4a9068f6cb1dce.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    added missing import of BinaryQuadraticModel required be DimodDecoder
```
Comment (arcondello) [inline releasenotes/notes/fix-dimod-importer-bqm-import-8c4a9068f6cb1dce.yaml:4]: ```Added missing import of ``BinaryQuadraticModel`` required by ``dimod.serialization.json.DimodDecoder`` ```

Comment (mhlr) [reply releasenotes/notes/fix-dimod-importer-bqm-import-8c4a9068f6cb1dce.yaml]: Done

**Review by arcondello — APPROVED** [review]

**Pull request merged**